atmo/keys/models.py

```diff
@@ -56,6 +56,5 @@ def prefix(self):
         return self.key.strip().split()[0]
 
     def save(self, *args, **kwargs):
-        if not self.fingerprint and self.key:
-            self.fingerprint = calculate_fingerprint(self.key)
+        self.fingerprint = calculate_fingerprint(self.key)
```

Comment:
Yes please 😺 . And could you also copy&paste the second part of [your comment above](https://github.com/mozilla/telemetry-analysis-service/pull/173#discussion_r99821763) into the commit message? It would be great for posterity